gulp-tasks/utils/build-browser-bundle.js

```diff
@@ -127,16 +128,23 @@ module.exports = (packagePath, buildType) => {
   logHelper.log(`    Namespace: ${logHelper.highlight(namespace)}`);
   logHelper.log(`    Filename: ${logHelper.highlight(outputFilename)}`);
 
+  const plugins = rollupHelper.getDefaultPlugins(buildType);
+
+  const banner = pkgJson.workbox.includeBabelHelpers ?
```

Comment:
I wonder if adding the `includeBabelHelpers` flag makes this more confusing than just hard coding workbox-core. My concern is that someone might start a new module and copy and paste workbox-core's package.json as a starter template and forget to remove this.

If we only want the helpers added in one place, I think it's fine to hard code when that should happen.

Reply:
You already need to edit the `package.json` `workbox` metadata if you're creating a new module, or else you'll end up reusing namespaces. So copy-and-pasting without making changes is already a non-starter.

I like the model of keeping build-specific metadata for a given package in a central place. Navigating through the various `gulp` tasks and their dependent modules is already a bit daunting without having to also check for hardcoded package-specific logic there.

Reply:
SGTM